### PR TITLE
incorporate new information about Compliance Masonry

### PR DESCRIPTION
**Pull request by afeld**
Closes #103.

### CODE REVIEW

File: _pages/ato/checklist.md [hunk 1]
```diff
@@ -43,6 +40,8 @@ You are welcome to ask any questions as comments in the issue or #infrastructure
         1. [ ] Add the issue-free scan report to [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing)
 1. [ ] Update relevant documentation, primarily the README
 1. [ ] Add a [System Security Plan](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
```
Comment (afeld) [inline _pages/ato/checklist.md]: @NoahKunin @jezhumble Is the existing [SSP template](https://pages.18f.gov/before-you-ship/ato/ssp/#template) still needed?

Comment (DavidEBest) [inline _pages/ato/checklist.md]: My understanding was that the existing SSP process will be used until the Infrastructure Leads program was fully ramped up.

Comment (afeld) [inline _pages/ato/checklist.md]: Okey doke. Good otherwise?

Comment (DavidEBest) [inline _pages/ato/checklist.md]: Yes, 👍  from me.

**Pull request merged**